tests/js-modules/modules.py

```diff
@@ -53,6 +53,79 @@ def test_module_import(network, args):
     return network
 
 
+@reqs.description("Test module bytecode caching")
+def test_bytecode_cache(network, args):
+    primary, _ = network.find_nodes()
+
+    bundle_dir = os.path.join(THIS_DIR, "basic-module-import")
+
+    LOG.info("Verifying that app works without bytecode cache")
+    proposal = network.consortium.set_js_app(
+        primary, bundle_dir, disable_bytecode_cache=True
+    )
+
+    raw_module_name = "/test_module.js".encode()
+    assert (
+        network.get_ledger_public_state_at(proposal.completed_seqno)
```

Comment:
I would like to avoid reading from the ledger in end to end tests unless we are specifically testing the ledger, because it's  unpredictably slow in CI (see #2674). Could you please add a bytecode size RPC, which will be useful to operators, and call that instead?

Reply:
Wouldn't a more general table statistics RPC make more sense?

Reply:
It's not been requested so far, and the implementation will either add substantial write overhead, or will come with a staggering execution cost unless we add zero-copy access (which is yet more work). Doing it just for the bytecode table, which is infrequently refreshed, is comparatively trivial and will make the testcases reliable.

So at this time, I think the answer is no.

Reply:
Done, I've named it `/node/js_metrics` but happy to rename. It returns `"bytecode_size": 123` and `"bytecode_used": true` where the latter is `false` if there is no bytecode or it needs re-compilation. See also the docs.